stl/inc/xutility

```diff
@@ -1988,18 +1986,6 @@ struct _Basic_const_iterator_category<_Iter> {
     using iterator_category = typename iterator_traits<_Iter>::iterator_category;
 };
 
-// TRANSITION, LLVM-55945: These are distinct concepts as a workaround
-template <class _Ty, class _Iter>
-concept _Bci_order = _Different_from<_Ty, basic_const_iterator<_Iter>> && random_access_iterator<_Iter>
-                  && totally_ordered_with<_Iter, _Ty>;
-
-template <class _Ty, class _Iter>
-concept _Bci_order_3way = _Bci_order<_Ty, _Iter> && three_way_comparable_with<_Iter, _Ty>;
-
-template <class _Ty, class _Iter>
-concept _Not_bci_order =
-    _Not_a_const_iterator<_Ty> && random_access_iterator<_Iter> && totally_ordered_with<_Iter, _Ty>;
-

```

Comment:
Having battled this overload set a couple of times, I think `_Bci_order` and `_Not_bci_order` improve readability quite a bit and would suggest keeping them. (I'm fine with expanding `_Bci_order_3way`, to be clear.) Is this just me, or do you think it's better to more closely reflect the Standard's wording?

Reply:
I'm fine with keeping `_Bci_order`. But the name `_Not_bci_order` is a bit confusing to me as it's not a negation of `_Bci_order`...

Reply:
Agreed, and I'm not motivated to devise better names. I'll let these go.